rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1770,32 +1777,35 @@ private void addSvpSpendTxSignatures(
         BtcECKey proposedFederatorPublicKey,
         List<byte[]> signatures
     ) {
+        // This flow should never be reached. There should always be a proposed federation if svpIsOngoing.
```

Comment:
I think this log was more accurate where it where

Reply:
Updated